ddtrace/debugging/_products/code_origin/span.py

```diff
@@ -1,4 +1,6 @@
+from ddtrace.settings._core import ValueSource
 from ddtrace.settings.code_origin import config
+from ddtrace.settings.dynamic_instrumentation import config as di_config
```

Comment:
Whilst Code Origin and DI are both under our remits, I wonder if it would be best to avoid cross-importing between different (sub-)products to avoid potential circular references. As an alternative, perhaps we can grab the DI configuration by querying the product manager, as in https://github.com/DataDog/dd-trace-py/blob/eff577329093ae7e4614b9aabecebcc19566a50c/ddtrace/internal/symbol_db/remoteconfig.py#L20. WDYT?